### ISSUE TITLE
Only 1 zone out of 4 visible in Homebridge

### ISSUE THREAD

**Issue by zakrabdea** [fjhorrillo/homebridge-airzone-cloud#38]
**Describe The Bug:**
I have 4 zones in my app Airzone Cloud and only 1 is visible in Homebridge and Homekit. The one that is visible works fine but I have 3 missing.

**Logs:**

```
[01/10/2023, 17:35:16] [AirzoneCloud] Logged in successfully as aeaes@outlook.fr
[01/10/2023, 17:35:17] [AirzoneCloud] Websocket connected
[01/10/2023, 17:35:17] [AirzoneCloud] The installation status was fully received
[01/10/2023, 17:35:17] [AirzoneCloud] Restoring existing accessory from cache: ZONE 01 (UUID: 7d181c2c-4031-4d3c-b2eb-fc3c50ae012f)
[01/10/2023, 17:35:17] [AirzoneCloud] Adding new accessory ZONE 02 (UUID: 3082241c-606a-400d-8687-d0698ef501cf)
[01/10/2023, 17:35:17] [TaHoma] TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)

```

**Plugin Config:**

```json
{
    "name": "AirzoneCloud",
    "system": "https://m.airzonecloud.com",
    "login": {
        "email": "xxxxxxx@outlook.fr",
        "password": "xxxxxxx"
    },
    "debug": false,
    "auto_off": true,
    "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.2 Safari/605.1.15",
    "platform": "AirzoneCloud"
}
```

**Screenshots:**
![Airzone Cloud](https://github.com/fjhorrillo/homebridge-airzone-cloud/assets/105398910/e26c9e38-f50d-4252-bbdd-df5619350437)
![Homekit](https://github.com/fjhorrillo/homebridge-airzone-cloud/assets/105398910/10d58324-3797-410b-8736-65701af043be)

**Environment:**

* **Plugin Version**: homebridge-airzone-cloud v0.3.6
* **Homebridge Version**: homebridge-config-ui-x v4.50.6
* **Node.js Version**: v18.18.0
* **NPM Version**: 9.8.1
* **Operating System**: Raspbian

**Comment by uxen-al** [comment]
Hi all, I have the same bug on starting homebridge 
I have 4 zone but no one is available 

[AirzoneCloud] Adding new accessory Théo (UUID: df78d184-d489-4249-a09b-d9df6313f8cc)
[07/10/2023, 13:13:58] TypeError: Cannot read properties of undefined (reading 'includes')
    at new AirzoneCloudPlatformAccessory (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)
    at AirzoneCloudHomebridgePlatform.registerDevice (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)
    at AirzoneCloudHomebridgePlatform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by pims31** [comment]
I have the same issue, 6 zones at home but only one showed on homebridge (the main one):

typeerror:%20Cannot%20read%20properties%20of%20undefined%20(reading%20'includes')%0Aat%20new%20AirzoneCloudPlatformAccessory%20(/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platformAccessory.ts:65:85)%0Aat%20AirzoneCloudHomebridgePlatform.registerDevice%20(/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:307:9)%0Aat%20AirzoneCloudHomebridgePlatform.discoverDevices%20(/var/lib/homebridge/node_modules/homebridge-airzone-cloud/src/platform.ts:150:20)%0Aat%20processTicksAndRejections%20(node:internal/process/task_queues:95:5)

**Comment by fjhorrillo** [comment]
Solved in v0.3.7, tested with node v20.9.0 and Homebridge 1.7.0 which are the last version
